src/lib/Zikula/Bundle/FormExtensionBundle/Resources/views/Form/bootstrap_3_zikula_admin_layout.html.twig

```diff
@@ -25,7 +25,13 @@ col-sm-9
             {{ form_widget(form) }}
             {{ form_errors(form) }}
             {% if help is defined %}
-                <em class="help-block small">{{ help }}</em>
+                {% if help is iterable %}
```

Comment:
what if help is not defined or empty? I had already changed these locally to `help|default`